app/styles/_global.scss

```diff
@@ -3,3 +3,48 @@
 #toast-container > div { // stylelint-disable-line selector-max-id
     opacity: 1;
 }
+
+@media (max-width: 480px) {
+    :root {
+        --ratio: 1.2;
+    }
+}
+@media (min-width: 480px) and (max-width: 768px) {
+    :root {
+        --ratio: 1.414;
+    }
+}
+@media (min-width: 768px) {
+    :root {
+        --ratio: 1.5;
+    }
+}
+
+:root {
+    --lineHeight: 1.5;
```

Comment:
Should this be used?

Reply:
It doesn't actually seem to do anything, so probably no.